client/app/scripts/liveblog-advertising/views/main.html

```diff
@@ -53,46 +54,134 @@ <h6 class="pull-left">
-                <button class="dropdown__toggle sd-create-btn"></button>
+                <button class="dropdown__toggle sd-create-btn" ng-click="openCollectionDialog();"></button>
             </div>
 	    </header>
+
+        <!-- LIST OF COLLECTIONS -->
+        <div class="advert-list-holder">
+            <div class="shadow-list-holder" sd-shadow>
+                <div class="flex-grid box wrap-items small-1 medium-3 large-4 xlarge-6">
+                    <div class="flex-item card-box" ng-repeat="collection in collections">
+                        <div class="card-box__header">
+                            <div class="dropdown" dropdown>
+                                <button class="dropdown__toggle" dropdown__toggle>
+                                    <i class="icon-dots-vertical"></i>
+                                </button>
+                                <ul class="dropdown__menu more-activity-menu pull-right">
+                                    <li><div class="dropdown__menu-label" translate>Actions</div></li>
+                                    <li class="divider"></li>
+                                    <li><button ng-click="openCollectionDialog(collection)" title="{{:: 'Edit Collection' | translate }}"><i class="icon-pencil"></i>{{:: 'Edit'| translate}}</button></li>
+                                
+                                    <li><button ng-click="removeCollection(collection, $index)" title="{{:: 'Remove collection'| translate }}"><i class="icon-trash"></i>{{:: 'Remove'| translate}}</button></li>
+                                </ul>
+                            </div>
+                            <div class="card-box__heading">{{ collection.name }}</div>
+                        </div>
+                        <div class="card-box__content">
+                            <ul class="card-box__content-list">
+                                <li ng-repeat="advert in adverts" ng-if="collectionHasAdvert(collection, advert)">
+                                    <h6 class="pull-left">
+                                        {{ advert.name }}
+                                    </h6>
+                                    <span class="state-label" ng-class="{ 'green-state': advert.type === 'Advertisement Local', 'yellow-state': advert.type === 'Advertisement Remote' }"}
+                                    >
+                                        {{ advert.type | translate }}
+                                    </span>
+                                </li>
+                                <li ng-if="collection.advertisements.length === 0" translate>No ads in this collection</li>
+                            </ul>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+
 	</div>
 </section>
 
+<!-- MODAL FOR ADDING EDITING ADVERTS -->
 <div sd-modal data-model="advertModalActive">
```

Comment:
I mean you can pass whatever scope variable you need to the child directive. I don't see that being too difficult